internal/app/app_darwin.go

```diff
@@ -52,9 +52,12 @@ func (a *App) install() error {
 		return errs.Wrap(err, "Could not create info file")
 	}
 
-	installDir, err := installation.ApplicationInstallPath()
-	if err != nil {
-		return errs.Wrap(err, "Could not get installation path")
+	installDir := os.Getenv("ACTIVESTATE_CLI_APPINSTALLDIR_OVERRIDE")
```

Comment:
I think using the added constant is appropriate here and other places where this magic value exists.